.github/workflows/demo_test.yml

```diff
@@ -400,3 +399,45 @@ jobs:
 
     - name: Check result
       run: docker exec python_support_test bash -c "cd /root/occlum/demos/python/occlum_instance; cat smvlight.dat"
+  # Redis test
+  Redis_support_test:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        submodules: true
+    - name: Create container
+      run: docker run -itd --name=redis_support_test -v $GITHUB_WORKSPACE:/root/occlum occlumbackup/occlum:latest-ubuntu18.04-python
```

Comment:
Use the original docker image like "run: docker run -itd --name=tflite_test -v $GITHUB_WORKSPACE:/root/occlum occlum/occlum:${{ env.OCCLUM_VERSION }}-ubuntu18.04" is better than using the python image. Because when you use the python image, each time when we do new release, we are not able to run this test case.